File: num.js
function printEvenOddAndSum(N1, N2) {
    let evenSum = 0;
    let oddSum = 0;

    for (let i = N1; i <= N2; i++) {
        if (i % 2 === 0) {
            console.log(i + " is even.");
            evenSum += i;
        } else {
            console.log(i + " is odd.");
            oddSum += i;
        }
    }

    console.log("Sum of even numbers: " + evenSum);
    console.log("Sum of odd numbers: " + oddSum);
}

// Example usage
printEvenOddAndSum(1, 10);

File: num.py
def print_even_odd_numbers(N1, N2):
    even_sum = 0
    odd_sum = 0
    
    for num in range(N1, N2+1):
        if num % 2 == 0:
            print(f"Even Number: {num}")
            even_sum += num
        else:
            print(f"Odd Number: {num}")
            odd_sum += num
    
    print(f"Sum of Even Numbers: {even_sum}")
    print(f"Sum of Odd Numbers: {odd_sum}")

# Define the range of numbers
N1 = 1
N2 = 10

# Call the function to print even and odd numbers and their sums
print_even_odd_numbers(N1, N2)
